packages/grafana-e2e/src/components/index.ts

```diff
@@ -0,0 +1,65 @@
+import { TestData } from '../pages/testdata';
+import { Panel } from '../pages/panel';
+import { EditPanel } from '../pages/editPanel';
+import { Graph } from '../pages/graph';
+import { componentFactory } from '../support';
+
+export const Components = {
+  DataSource: {
+    TestData,
+  },
+  Panels: {
+    Panel,
+    EditPanel,
+    Visualization: {
+      Graph,
+    },
+  },
+  Drawer: {
+    General: componentFactory({
+      selectors: {
+        title: (title: string) => `Panel inspector title ${title}`,
+        expand: 'Panel inspector expand',
+        contract: 'Panel inspector contract',
+        close: 'Panel inspector close',
+        rcContentWrapper: () => '.drawer-content-wrapper',
+      },
+    }),
+  },
+  PanelInspectData: componentFactory({
+    selectors: {
+      content: 'Panel inspector Data content',
+    },
+  }),
+  PanelInspectStats: componentFactory({
+    selectors: {
+      content: 'Panel inspector Stats content',
+    },
+  }),
+  PanelInspectJSON: componentFactory({
+    selectors: {
+      content: 'Panel inspector JSON content',
+    },
+  }),
+  PanelInspectQuery: componentFactory({
+    selectors: {
+      content: 'Panel inspector Query content',
+    },
+  }),
```

Comment:
I do like the flatter strucuture but maybe these should be nested under a PanelInspector { StatsTab,  DataTab: etc}

Reply:
Yes, I'm moving away from those deep nested paths, but I'll see what I can do.